Screenbox/Pages/MainPage.xaml

```diff
@@ -26,11 +26,8 @@
             <Thickness x:Key="AppTitleMarginMinimal">98,0,0,0</Thickness>
 
             <DataTemplate x:Key="SearchAutoSuggestBoxItemTemplate">
-                <StackPanel Orientation="Horizontal">
-                    <FontIcon
-                        Margin="0,0,8,0"
-                        FontSize="16"
```

Comment:
The font size is necessary to make the icon smaller. I see you already have a resource for `FontIcon` in `App.xaml` but it may not work if the icon is in a flyout or a dialog since they use a different visual root from the main application.

old:
![image](https://github.com/huynhsontung/Screenbox/assets/31434093/9db5dd80-b534-428a-9e29-e4629736caf1)

new:
![image](https://github.com/huynhsontung/Screenbox/assets/31434093/55331e56-5462-4c47-afc8-8cf1dd1db3a8)

Reply:
Side note you can see the 16 font size is sharper than the default size on a 1920x1080 screen, 100% scaling.